### PR TITLE
Added tests for URLs.

### PR DESCRIPTION
**Pull request by Wolverin-e**
added tests for URLs in apps company, main, student and in spc_django.

### CODE REVIEW

File: src/company/tests/test_urls.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    def test_job_offer_url_resolves(self):
+        job_offer = JobAdvertisement.objects.create(active=True, company=self.test_company, designation=' ', description=' ',
+                                                    tentative_join_date=date(2030, 10, 23), tentative_job_location=' ',
+                                                    ctc='23.5', bond=False, resume_required=False, aptitude_test_required=False,
+                                                    group_discussion_required=False, medical_test_required=False, min_gpa=3.5)
+        url = reverse('company:job-offer', kwargs={'id': job_offer.id})
+        self.assertEquals(resolve(url).func.view_class, JobOfferView)
+
+
+class TestCompanyUrlsResolve1(SimpleTestCase):
```
Comment (anshulahuja98) [inline src/company/tests/test_urls.py]: Remove the `1`

File: src/spc_django/tests.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        url = reverse('password_reset_complete')
+        self.assertAlmostEquals(resolve(url).func, password_reset_complete)
+
+
+class TestAccountsUrlsResolves1(TestCase):
+
+    def setUp(self):
+        self.client = Client()
+        self.login_url = reverse('login')
+        self.test_user = User.objects.create_user(username='testuser', password='A lph a Be ta 124')
```
Comment (anshulahuja98) [inline src/spc_django/tests.py]: How does this password work?

**Comment by anshulahuja98** [comment]
Update travis to run the tests

File: src/spc_django/tests.py (hunk 2 above)
Comment (Wolverin-e) [reply src/spc_django/tests.py]: Changed!

**Pull request merged**